.github/tests/application/containers/syslog/syslog.Dockerfile

```
@@ -0,0 +1,11 @@
+FROM alpine:3
+
+RUN apk add --no-cache rsyslog
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)